packages/blade/src/_helpers/storybook/StoryPageWrapper.tsx

```diff
@@ -42,36 +73,52 @@ const StoryPageWrapper = (props: StoryPageWrapperTypes): React.ReactElement => {
   ]);
 
   return (
-    <>
+    <WithGlobalStyles>
       <Title>{props.componentName}</Title>
       <Subtitle>{props.componentDescription}</Subtitle>
       {props.note ? <Description markdown={`> **Note** <br/>${props.note}`} /> : null}
       {figmaURL !== '#' ? (
         <FigmaEmbed src={figmaURL} title={`${props.componentName} Figma Designs`} />
       ) : null}
-      {props.children}
+      <BladeProvider themeTokens={paymentTheme}>{props.children}</BladeProvider>
```

Comment:
Hard-coding payment theme? Shouldn't we have a mechanism to switch it for overall storybook too?

It kinda breaks when I switch theme to banking

<img width="1102" alt="Screenshot 2023-03-13 at 3 35 08 PM" src="https://user-images.githubusercontent.com/24487274/224670305-c7f179b3-4ae3-4135-94b8-a0e76dfac6b8.png">

Reply:
> Hard-coding payment theme? Shouldn't we have a mechanism to switch it for overall storybook too?

This is what we tried during the last theme change task and figured out that its pain in storybook. There are components in storybook which come directly from storybook and only accept static string for colors so even if we put switch, it will switch for some parts and won't switch for some. 

This breakpoints documentation in particular is breaking because we didn't hardcode colorScheme for it. Let me do that.

Reply:
done in upcoming commit